src/client/components/pages/parts/admin-panel-search-results.tsx

```diff
@@ -143,9 +140,9 @@ class AdminPanelSearchResults extends React.Component<AdminPanelSearchResultsPro
 				>
 					<thead>
 						<tr>
-							<th className="col-md-5">Name</th>
-							<th className="col-md-5">Privileges</th>
-							<th className="col-md-5"/>
+							<th width="42%">Name</th>
+							<th width="42%">Privileges</th>
+							<th width="42%"/>
```

Comment:
I haven't checked the final output, but using my quick maths power I deduct we have 126% total.
I guess the first two columns take 42% each and the last one gets what's left (16%)?
In which case you can probably just leave out the width for the last column header.

Reply:
haha! my bad. I thought the 42% means that it will take up a maximum of 42% width of the total container depending upon the contents.

Reply:
Do check the result, I haven't run the code so this is just a hunch

Reply:
at first glance, it looked alright. But I think it is proportionally scaling down the width percentages depending upon the content in each column, if the sum is greater than 100. If the sum is 100, then it keeps the proportion constant.